### PR TITLE
Backup ARM and Classic IaaSVMs to Recovery Services Vault

### PR DESCRIPTION
**Pull request by nilaydshah**
Backup ARM and Classic IaaSVMs to Recovery Services Vault. 
This template assumes IaaSVMs, Recovery Services Vault and Backup policy are already existing. The template configures protection for existing IaaSVMs.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@nilaydshah__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (nilsha). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by nilaydshah** [comment]
@singhkay CI build for this template is failing due to consideration of existing resources in the template. Please review this.

File: 101-recovery-services-backup-iaasvm/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        "existingBackupPolicyName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of existing Backup Policy in same Recovery Services Vault."
+            }
+        }
+    },
+    "variables": {
+        "backupFabric": "Azure",
+        "protectedItemType": "Microsoft.ClassicCompute/virtualMachines"
```
Comment (singhkay) [inline 101-recovery-services-backup-iaasvm/azuredeploy.json]: Does this only work with classic VMs?

File: 101-recovery-services-backup-iaasvm/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Backup ARM and Classic IaaSVMs to Recovery Services Vault
```
Comment (singhkay) [inline 101-recovery-services-backup-iaasvm/README.md]: Let's remove the "IaaS" before the vm. Vm alone should be enough

File: 101-recovery-services-backup-iaasvm/azuredeploy.json (hunk 1 above)
Comment (nilaydshah) [inline 101-recovery-services-backup-iaasvm/azuredeploy.json]: No it works for both ARM and classic. RP identifies this based on existingProtectionContainers and existingProtectableItems value

**Comment by nilaydshah** [comment]
@singhkay updated folder name

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**